client/src/fluid/FluidAutocomplete.ml

```diff
@@ -328,12 +328,32 @@ let secretToACItem (s : SecretTypes.t) : fluidAutocompleteItem =
   let asDval = DStr (Util.obscureString s.secretValue) in
   FACVariable (s.secretName, Some asDval)
 
+let isInQuery (tl : toplevel) ti =
```

Comment:
There's a possibility that this will be slow and cause problems in rendering, but I think it's fine (typically the things that are slow do an AST traversal per token, and I think this just does one per keystroke, which is probably fine).